README.md

```diff
@@ -183,4 +183,4 @@ Some design principles which should be considered:
 [cargo-llvm-cov]: https://github.com/taiki-e/cargo-llvm-cov
 [FFI]: ffi/
 [Arrow]: https://arrow.apache.org/rust/arrow/index.html
-[Tokio]: https://tokio.rs/
+[Tokio]: https://tokio.rs/
```

Comment:
out of curiosity, what changed here?

Reply:
I think the file didn't have a newline at EOF before, and now it does?